src/pages/help/Help.tsx

```diff
@@ -3,45 +3,49 @@ import { IonCol, IonContent, IonIcon, IonPage } from "@ionic/react";
 import "./Help.css";
 import { svgs } from "../../assets/svgs";
 import ActionAnchorLink from "../../components/Action/ActionAnchorLink";
+import {useTranslation} from "react-i18next";
+import parse from "html-react-parser";
 
 const Tab1: React.FC = () => {
+  const { t } = useTranslation('help');
   return (
     <IonPage>
       <IonContent className="ion-padding">
         <section className="search-title">
           <IonCol size="1" class="icon ion-no-padding">
             <IonIcon size="small" icon={svgs.HELP}></IonIcon>
           </IonCol>
-          <h2>About</h2>
+          <h2>{t('about_heading', 'About')}</h2>
         </section>
 
         <hr className="header-divider" />
-        <p>
-          The Constitution App is developed by the{" "}
+        {parse(t('app_description', `
+          <p>
+            The Constitution App is developed by the&nbsp;
```

Comment:
I'm not sure why the `{" "}` was there in the first place, but I'm also not sure that they need to be replaced with non-breaking spaces. Is a normal space suitable?

Reply:
Normally we'd do that so it doesn't line wrap at that point, but I don't think it matters?

Reply:
> 

yea lets remove the nbsp. its not actually needed

Reply:
yea, nbsp or `{" "}`is not necessary for these `p` tags